### DISCUSSION TITLE
Incorrect mime_type in database

### DISCUSSION THREAD

**Discussion by JamesPJ** [spatie/laravel-medialibrary#3241] [Q&A]
Laravel 5.8
spatie/laravel-medialibrary 7.19

 ```
$model->addMedia(storage_path('tmp/uploads/' . $uid . '/' . $file))
                        ->preservingOriginal()
                        ->withCustomProperties(['source' => 'local'])
                        ->toMediaCollection(config('app.name').'/media/' . $folder . '/', $drive);
```
This is how I'm uploading 

I'm uploading `mp3` file but, `addMedia` setting the `mimetype` as `video/3gpp`. Let me know any way I can manually pass proper mimetype to `addMedia`.

This is what will be the upload path

`$model->addMedia($file)->toMediaCollection(config('app.name').'/voice/','s3');`

What I'm referring is something like this

`$model->addMedia($file, 'audio/mpeg')->toMediaCollection(config('app.name').'/voice/','s3');`

**Comment by francoism90** [comment]
If I'm not mistaken, you can append WithAttributes, I do this to overrule the user id.

The mimetype should be correct however, you can use tools like mediainfo to check the content type.
